wb-codemod/src/wb-codemod.ts

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-console */
+import path from "path";
+import chalk from "chalk";
+import jscodeshift from "jscodeshift/src/Runner";
+
+type RunOptions = {
+    /**
+     * Whether to dry run the codemod.
+     *
+     * If true, the codemod will not be applied in the files, but the
+     * transformed code will be printed.
+     */
+    dryRun?: boolean;
+    /**
+     * Whether to print the output of the codemod.
+     */
+    print?: boolean;
+};
+
+/**
+ * Runs jscodeshift on the given transform file (codemod) and file paths.
+ */
+export async function run(
+    transformFileName: string,
+    filePaths: Array<string>,
+    options: RunOptions,
+) {
+    const jsCodemodsDir = path.resolve(__dirname, "../transforms");
+    // Transform path
+    const transformFile = path.join(jsCodemodsDir, transformFileName + ".ts");
+
+    console.log(
+        chalk.cyan(`Transforming ${transformFileName} in: ${filePaths}...`),
+    );
+
+    try {
+        const result = await jscodeshift.run(transformFile, filePaths, {
+            dry: options.dryRun || false,
+            print: options.print || false,
+            verbose: 2,
+            babel: true,
+            extensions: "js,jsx,ts,tsx",
+            parser: "tsx",
+            runInBand: true,
```

Comment:
Why are you doing this? IMO, this makes it a lot slower. You could make this `true` if `dryRun` is true, instead?

Reply:
Good point! I'll change that.